packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -967,7 +967,7 @@ class ClipRect extends SingleChildRenderObjectWidget {
 ///  * [ClipRect], for more efficient clips without rounded corners.
```

Comment:
There are some doc changes in this file that are unrelated to this change. Can you remove them? We can always contribute them in another separate PR.